### PR TITLE
do_build.sh: Add meta-virtualization layer handling.

### PR DESCRIPTION
**Pull request by eric-ch**
do_build is deprecated, it is recommended to use the build-scripts. Yet some people might still have setups using the legacy script.

### CODE REVIEW

File: setup_build [hunk 1]
```diff
@@ -68,6 +68,9 @@ META_OPENXT_OCAML_PLATFORM_TAG=master
 
 META_OPENXT_HASKELL_PLATFORM_REPO=https://github.com/OpenXT/meta-openxt-haskell--platform.git
 META_OPENXT_HASKELL_PLATFORM_TAG=master
+
+META_OPENXT_HASKELL_PLATFORM_REPO=git://git.yoctoproject.org/meta-virtualization
```
Comment (jean-edouard) [inline setup_build]: Bad copy paste!

File: setup_build [hunk 2]
```diff
@@ -68,6 +68,9 @@ META_OPENXT_OCAML_PLATFORM_TAG=master
 
 META_OPENXT_HASKELL_PLATFORM_REPO=https://github.com/OpenXT/meta-openxt-haskell--platform.git
```
Comment (jean-edouard) [inline setup_build]: Two dashes?

Comment (eric-ch) [inline setup_build]: Apparently a typo that slipped through, fixing...

File: setup_build (hunk 1 above)
Comment (eric-ch) [inline setup_build]: Indeed, fixed.

**Comment by jean-edouard** [comment]
Merging soon.

**Pull request merged**